### DISCUSSION TITLE
genomic DNA contamination

### DISCUSSION THREAD

**Discussion by AnikoSabo** [gagneurlab/drop#552] [Q&A]
Hi, 
I have a dataset of about 350 samples in which maybe about 5% of samples have some level of genomic contamination. I suspect genomic contamination as the % of intergenic reads is quite high in some samples, and looks like outliers from the rest (<5% intergenic reads in most samples). How sensitive is DROP to genomic DNA contamination in a dataset? I can remove these samples, but was wondering what would be the effect on the analysis if I included them, and if I can somehow "rescue" these samples. Also the bam for these samples are quite large (20-30GB), so they are difficult to run with a reasonable memory amount (180GB, 5 cores). Any ideas would be appreciated. Thanks

**Comment by vyepez88** [comment]
Hi, DROP is intended to detect outliers, it is expected (and we have seen in many cohorts) that contaminated samples will have lots of aberrations. If you really want to rescue them, I would create 2 groups, one with and one without the contaminated samples.
For the AE module, DROP only counts reads assigned to a gene, so the intergenic reads will not an effect, however it could be that the distribution of reads assigned to exons is also different.
For the AS module, it could be that split reads are counted from exonic to the intergenic regions, thus creating lots of splicing outliers. Overall I would be careful with them.
Regarding the quite large, not sure, we have ideas on how to make DROP counting more efficient, but nothing implemented yet.
